modules/apps/object/object-test/src/testFunctional/tests/ObjectFieldsDateAndTime.testcase

```diff
@@ -149,5 +149,59 @@ definition {
 		}
 	}
 
+	@description = "LPS-143068 - Verify if is possible to edit DateTime on entries from Custom Object"
+	@priority = 5
+	test CanUpdateEntry {
+		property portal.acceptance = "true";
+
+		task ("Given a custom object with a date and time field as UTC") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object 186247",
+				objectName = "CustomObject186247",
+				pluralLabelName = "Custom Objects 186247");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject186247");
+
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectCustomObject(label = "Custom Object 186247");
+
+			ObjectAdmin.goToFieldsTab();
+
+			ObjectAdmin.addObjectFieldViaUI(
+				fieldLabel = "Custom Date Time",
+				fieldTimeStorage = "Convert to UTC",
+				fieldType = "Date and Time");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject186247");
+		}
+
+		task ("And given an object entry added by user") {
+			ObjectAdmin.addObjectSingleFieldEntryViaAPI(
+				fieldName = "customDateTime",
+				objectName = "CustomObject186247",
+				value = "2023-06-01T12:00:00.000Z");
+
+			ObjectAdmin.goToCustomObject(objectName = "CustomObject186247");
+		}
+
+		task ("When the user update the entry") {
```

Comment:
`When the user updates the entry`

Please pay careful attention to the wording on the spreadsheet